packages/material-ui-icons/package.json

```diff
@@ -42,7 +42,7 @@
   },
   "dependencies": {
     "@babel/runtime": "7.0.0",
-    "recompose": "^0.29.0"
+    "recompose": "0.28.0 - 0.30.0"
```

Comment:
@oliviertassinari Why was this necessary?

Reply:
It reduce the dependency duplication in peaple bundles.

Reply:
@oliviertassinari  Curious. I  understand that it reduces duplication if we reduce the lower bound. But adding an upper bound should potentially increase it if users pin their own version to `^0.31`, no?

Reply:
Right. It's our only 0.x dependency. Still used in our docs though which makes me wonder why `eslint` is not reporting that we're using implicit dependencies.